Tests/NIOTests/HappyEyeballsTest.swift

```diff
@@ -1152,6 +1152,104 @@ public class HappyEyeballsTest : XCTestCase {
             XCTFail("Got unexpected error: \(String(describing: channelFuture.getError()))")
         }
     }
+
+    func testCancellationSyncWithConnectDelay() throws {
+        var channels: [Channel] = []
+        defer {
+            channels.finishAll()
+        }
+
+        let (eyeballer, resolver, loop) = buildEyeballer(host: "example.com", port: 80, connectTimeout: .milliseconds(250)) {
```

Comment:
@Lukasa maybe we just want to use `swiftnio.io` for the hostname everywhere tho ?

Reply:
I think it doesn't really matter?

Reply:
Yeah sure it not matters in this test but if we have a domain name we can also just use it and not refer to some random ones. Feel free to ignore tho, I don’t say we need to do it but I think knit is a good thing in general

Reply:
Actually, an `example.com` is just OK for this purpose, see [rfc2606](https://tools.ietf.org/html/rfc2606).

Reply:
@vlm TIL ... Thanks!